openedx/core/djangoapps/user_authn/views/registration_form.py

```diff
@@ -1007,6 +1008,12 @@ def _add_terms_of_service_field(self, form_desc, required=True):
             ),
             tos_link_end=HTML("</a>"),
         )
+        label_with_terms_of_service_and_privacy_policy = run_extension_point(
+            'PEARSON_CORE_REGISTRATION_FORM_MODULE',
+            platform_name=configuration_helpers.get_value("PLATFORM_NAME", settings.PLATFORM_NAME),
+        )
+        if label_with_terms_of_service_and_privacy_policy:
+            label = label_with_terms_of_service_and_privacy_policy
```

Comment:
```suggestion
        label = label_with_terms_of_service_and_privacy_policy if label_with_terms_of_service_and_privacy_policy else label
```
What do you think? @anfbermudezme